.github/workflows/try-runtime-mainnet.yml

```diff
@@ -8,9 +8,9 @@ on:
         description: The WebSocket url of the reference node
         default: wss://ws.calamari.systems:443
         required: true
-      chain_spec:
-        description: The chain spec of the local node
-        default: calamari-dev
+      runtime:
+        description: The runtime of the local node - use `manta` or `calamari` only.
+        default: calamari
```

Comment:
let's try to use `manta` as a default setting for everything from now on

Reply:
Good point